modules/nf-core/hamronization/rgi/tests/main.nf.test

```diff
@@ -9,22 +9,50 @@ nextflow_process {
     tag "hamronization"
     tag "hamronization/rgi"
     tag "rgi/main"
+    tag "rgi/cardannotation"
+    tag "untar"
 
     setup {
+        run("UNTAR") {
+            script "modules/nf-core/untar/main.nf"
+            process {
+            """
+            file('https://card.mcmaster.ca/latest/data', checkIfExists: true).copyTo('data.tar.gz')
```

Comment:
Does this work!? When I download it it's a bz2 file?!

Reply:
Interesting, with `wget` (which is apparently also internally used for `file('<URL>')` only a file named `data` is created without suffix. That made it impossible for the `untar` module to extract because it wants to create a folder with the basename. Thus I changed the ending, apparently the compression algo is not relevant here. Lucky for us, because the `untar` module accepts only `tar.gz` files at the moment, not `.tar.bz2` (just tested).